lua/damagelogs/client/tabs/rdm_manager.lua

```diff
@@ -336,13 +346,17 @@ local function TakeAction()
         slaynr2:AddOption(TTTLogTranslate(GetDMGLogLang, "TheVictim") .. " (" .. report.victim_nick .. ")", function()
             if IsValid(victim) then
                 if ulx then
-                    RunConsoleCommand("ulx", mode == 1 and "aslayid" or "ajailid", report.attacker, "0")
-                else
+                    RunConsoleCommand("ulx", mode == 1 and "aslay" or "ajail", victim:Nick(), "0")
```

Comment:
Can this go back to `aslayid`/`ajailid`?

I remember this command having problems in the past with players that had weird characters in their names.

Reply:
done